test/recipes/95-test_external_rpki-client-portable_data/rpki-client-portable.sh

```diff
@@ -11,9 +11,8 @@
 LDFLAGS="-L`pwd`/$BLDTOP -Wl,-rpath,`pwd`/$BLDTOP"
 CFLAGS="-I`pwd`/$BLDTOP/include -I`pwd`/$SRCTOP/include"
 
-cd $SRCTOP/rpki-client-portable
-
-./autogen.sh
+wget -O $RPKI_TARBALL $RPKI_DOWNLOAD_URL && tar xzf $RPKI_TARBALL
```

Comment:
Just out of curiosity, why not just set the HEAD ref of the submodule to the tag that we want to track?

Reply:
the github mirror of rpki-client does not use tags.

Reply:
It appears to from my view:
https://github.com/rpki-client/rpki-client-portable/tags

Or is there more to switching from the github tree to the cdn you are using above?

Reply:
I was told by @botovq to use release packages. So I just went that way.

Reply:
ok, good enough for me.  Thanks!